pkg/scheduler/scheduler.go

```diff
@@ -187,127 +139,18 @@ func (s *Scheduler) Run(ctx context.Context) {
 	}
 
 	s.informerFactory.Start(stopCh)
-	if !cache.WaitForCacheSync(stopCh, s.bindingInformer.HasSynced) {
-		return
-	}
+	s.informerFactory.WaitForCacheSync(stopCh)
```

Comment:
If there is no cacheSyncs to wait, does this line still need?

Reply:
if there is no cr object to wait， WaitForCacheSync will directly return

Reply:
ok, I got it.